app/templates/tasks.hbs

```diff
@@ -0,0 +1,21 @@
+{{page-title "My Tasks"}}
+
+<div  class="tasks-screen_wrapper">
+  <SideBar
+    @showDropDown={{this.showDropDown}}
+    @toggleDropDown={{this.toggleDropDown}}
+    @toggleActiveTasks={{this.toggleActiveTasks}}
+    @toggleBlockedTasks={{this.toggleBlockedTasks}}
+    @togglePendingTasks={{this.togglePendingTasks}}
```

Comment:
Nit: Sounds like we are using too many toggles. How can we simplify this?

Reply:
Done